geoscilabs/gravity/gravityDike.py

```diff
@@ -0,0 +1,324 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.contour as ctr
+import scipy.io
+import copy
+from math import pi, tan, cos, acos, log, sin
+from ipywidgets import (
+    interactive,
+    IntSlider,
+    widget,
+    FloatText,
+    FloatSlider,
+    ToggleButton,
+    VBox,
+    HBox,
+    Output,
+    interactive_output,
+    Layout,
+)
+
+plotdata = []  # Storage the data that will print on the picture
+colorList = [
+    "red",
+    "blue",
+    "green",
+    "orange",
+    "black",
+    "pink",
+    "brown",
+    "deepskyblue",
+    "darkkhaki",
+    "fuchsia",
+    "midnightblue",
+    "yellow",
+    "gold",
+    "lime",
+]  # The Colors
+index = 0
+currentResult = []
+
+# The main function and draw the first table
+def drawfunction(delta_rho, z1, z2, b, beta, stationSpacing, B):

```

Comment:
It would be good to have documentation such as:
`def forward(nD,delta_rho):
"""
   Parameters
        ----------
        nD: int
            number of data
        delta_rho : numpy.array
            model of density contrasts

        Returns
        -------
        np.array
            data, the forward data
"""

But maybe not required for this PR. What's your opinion on it @lheagy ?

Reply:
It would be nice and very much appreciated, but I don't think we can ask that it be required for this PR. We haven't set that precedent within the codebase yet.

Reply:
Sounds good!